src/components/sky-nav/sky-nav.stories.mdx

```diff
@@ -1,47 +1,72 @@
+
+- By default, menu items are hidden until toggled. They are revealed _above_ the brand and toggle, as if looking upward toward the sky.
+- In wider viewports, menu items are visible by default. They are displayed on multiple rows as needed.
+- At even wider viewports, the menu items are displayed in a single row.
+
+The Sky Nav is intended for use with our [our dark theme](/?path=/docs/themes--dark):
 
-We are tentatively using the same template properties as [Timber Menus](https://timber.github.io/docs/guides/menus/).
+<Preview>
+  <Story name="Dark" height="200px" parameters={{ themes: { disable: true } }}>
+    {template({ class: 't-dark', include_main_demo: true, menu })}
+  </Story>
+</Preview>
+
+But it works without a theme, too, just in case:
 
 <Preview>
-  <Story name="Markup (WIP)">{template({ menu })}</Story>
+  <Story name="Light" height="200px">
+    {template({ include_main_demo: true, menu })}
+  </Story>
 </Preview>
+
+## Elements
+
+- `c-sky-nav`: A containing `<div>` element.
+- `c-sky-nav__skip`: A link to the main content area, visibly hidden until focus. Works great with [our `c-button` pattern](/?path=/docs/components-button--elements).
+- `c-sky-nav__content`: A `<header>` element representing the actual menu contents.
+- `c-sky-nav__masthead`: Containing element for the logo and menu toggle.
+- `c-sky-nav__logo`: A link to the homepage containing our logo as an inline SVG.
+- `c-sky-nav__logo-object`: The `<svg>` element itself.
+- `c-sky-nav__logo-outline`: A path within the SVG representing the focus outline.
+- `c-sky-nav__menu-toggle`: Control that toggles the menu visibility on small screens. Another great fit for `c-button`.
+- `c-sky-nav__menu`: A `<nav>` element containing the list of menu items.
+- `c-sky-nav__menu-items`: A `<ul>` containing menu items.
+- `c-sky-nav__menu-item`: An individual `<li>` containing a menu action.
+- `c-sky-nav__menu-action`: Represents a single interactive option within the menu, most likely a link.
+- `c-sky-nav_-menu-label`: Wrapper for the action's text content. Used as a hook for focus styles.
+
+## Accessibility
+
+As the primary navigation element for our site, it's imperative we think through the accessibility implications of this component. Here are a few specific considerations we've made so far:
+
+- All interactive elements contain specially designed focus outlines. This is especially important since some browser's default outlines virtually disappear against our blue!
+- The "skip to main content" link precedes the header and navigation elements and can be used to skip them entirely. It is visually hidden until focus but always detectable by screen readers.
+- The `<header>` element includes a `role` attribute of `banner` because [it is not a direct child of the `<body>` element](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Banner_Role).
```

Comment:
> Assistive technologies can identify the main `header` element of a page as the `banner` if is a descendant of the `body` element, and not nested within an `article`, `aside`, `main`, `nav` or `section` subsection.

My reading of this is that "descendant" does not mean "direct child."

Will this be nested within an `article`, `aside`, `main`, `nav` or `section`?

If not, then I think we don't need the `role` after all.

Reply:
It's nested ina  `<div>` as part of this pattern.

Reply:
I think you're right, @spaceninja. That distinction is also repeated at the [W3 ARIA Practices page](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/banner.html).

I will make the following updates:
- Remove `role="banner"`
- Rewrite this note to be more of an "FYI, if for some strange reason you are including this within a sectioning element..."
- Replace the source link for that note with the W3 one that's a tad more concise.